playbooks/dataiku.yml

```diff
@@ -1,40 +1,38 @@
 - hosts: 127.0.0.1
   connection: local
-  become_user: ubuntu
   gather_facts: no
-  roles:
-    - python3.6-base
   vars_files:
     - ./group_vars/dataiku
 
   tasks:
-    - name: I am confused b
-      command: 'whoami'
-      register: myidentity
-
-    - name: my secret identity
-      debug:
-        msg: '{{ myidentity.stdout }}'
-
-    - name: install default-jdk
-      apt:
-        name: openjdk-11-jdk
+    - name: Increase system limits as required by DSS
```

Comment:
No feedback but just for my knowledge: why are you setting the pam_limits? Do I understand correctly that the pam_limits are set low and therefore you wish to increase them? What is the element that limits them in the first place?